hugegraph-core/src/main/java/com/baidu/hugegraph/task/HugeTask.java

```diff
@@ -258,6 +291,7 @@ protected void status(TaskStatus status) {
         this.status = status;
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
unnecessary

Reply:
line 329 needs it.
for (Id id : (Set<Id>) value) {